api/src/services/dwc-service.ts

```diff
@@ -90,4 +70,86 @@ export class DwCService extends DBService {
     // Apply patch operations
     return jsonpatch.applyPatch(jsonObject, patchOperations).newDocument;
   }
+
+  /**
+   * Run all Decoration functions on DWCA Source Data
+   *
+   * @param {number} occurrenceSubmissionId
+   * @return {*}  {Promise<boolean>}
+   * @memberof DwCService
+   */
+  async decorateDWCASourceData(occurrenceSubmissionId: number): Promise<boolean> {
+    const occurrenceService = new OccurrenceService(this.connection);
+
+    const submission = await occurrenceService.getOccurrenceSubmission(occurrenceSubmissionId);
+    const jsonObject = submission.darwin_core_source;
+
+    let newJson = await this.decorateLatLong(jsonObject);
```

Comment:
Can you make a function that decorates the json object ie ```decorateDWCSource``` and have that function call the ```decorateLatLong``` and ```decorateTaxonIDs```

Reply:
I believe the function in comment is that function, ie `decorateDWCASourceData`